src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
    */
-  HOM_ALT,
+  union { null, int } alternateReverseReadDepth = null;
 
   /**
-   The genotype could not be called at this site.
+   Per-sample total reference read depth. VCF genotype reserved key AD, Number=R, split for
+   multi-allelic sites into single integer values for the reference allele (referenceReadDepth,
+   this field) and the alternate allele (alternateReadDepth).
    */
-  NO_CALL
-}
-
-/**
- This record represents all stats that, inside a VCF, are stored outside of the
- sample but are computed based on the samples. For instance, MAPQ0 is an aggregate
- stat computed from all samples and stored inside the INFO line.
- */
-record VariantCallingAnnotations {
+  union { null, int } referenceReadDepth = null;
 
   /**
-   True if filters were applied for this genotype call. FORMAT field "FT" any value other
-   than the missing value.
+   Per-sample forward strand reference read depth. VCF genotype reserved key ADF, Number=R,
+   split for multi-allelic sites into single integer values for the reference allele
+   (referenceForwardReadDepth, this field) and the alternate allele (alternateForwardReadDepth).
    */
-  union { null, boolean } filtersApplied = null;
+  union { null, int } referenceForwardReadDepth = null;
 
   /**
-   True if all filters for this genotype call passed. FORMAT field "FT" value PASS.
+   Per-sample reverse strand reference read depth. VCF genotype reserved key ADR, Number=R,
+   split for multi-allelic sites into single integer values for the reference allele
+   (referenceReverseReadDepth, this field) and the alternate allele (alternateReverseReadDepth).
    */
-  union { null, boolean } filtersPassed = null;
+  union { null, int } referenceReverseReadDepth = null;
 
+  // DP Number=1 Type=Integer
   /**
-   Zero or more filters that failed for this genotype call from FORMAT field "FT".
+   Read depth at this position for this sample. VCF genotype field reserved key DP.
    */
-  array<string> filtersFailed = [];
+  union { null, int } readDepth = null;
 
+  // EC Number=A Type=Integer
+  // Comma separated list of expected alternate allele counts for each alternate allele in the same
+  // order as listed in the ALT field. Typically used in association analyses.
   /**
-   True if the reads covering this site were randomly downsampled to reduce coverage.
+   The expected dosage of the alternate allele in this sample. VCF genotype reserved key EC,
+   Number=A, split for multi-allelic sites into a single float value.
    */
-  union { null, boolean } downsampled = null;
+  union { null, float } expectedAlleleDosage = null; // or expectedAlleleCount? see Variant.alleleCount
 
+  // GL Number=G Type=Float
   /**
-   The Wilcoxon rank-sum test statistic of the base quality scores. The base quality
-   scores are separated by whether or not the base supports the reference or the
-   alternate allele.
+   Genotype likelihoods comprised of floating point log10-scaled likelihoods for all possible
```

Comment:
That doc text comes from VCF spec.  The phrase `[the set of alleles defined in the REF and ALT fields]` needs expansion to explain how the values are split for multi-allelic sites.